bugsnag-plugin-android-ndk/src/main/jni/metadata.c

```diff
@@ -285,27 +284,17 @@ void bsg_populate_device_data(JNIEnv *env, bsg_jni_cache *jni_cache,
   bsg_copy_map_value_string(env, jni_cache, data, "locale",
                             event->device.locale,
                             sizeof(event->device.locale));
-  bsg_copy_map_value_string(env, jni_cache, data, "locationStatus",
-                            event->device.location_status,
-                            sizeof(event->device.location_status));
-  bsg_copy_map_value_string(env, jni_cache, data, "networkAccess",
-                            event->device.network_access,
-                            sizeof(event->device.network_access));
+
   bsg_copy_map_value_string(env, jni_cache, data, "osVersion",
                             event->device.os_version,
                             sizeof(event->device.os_version));
-  bsg_copy_map_value_string(env, jni_cache, data, "screenResolution",
-                            event->device.screen_resolution,
-                            sizeof(event->device.screen_resolution));
-  event->device.emulator =
-      bsg_get_map_value_bool(env, jni_cache, data, "emulator");
+
+  bsg_strcpy(event->device.os_name, bsg_os_name());
   event->device.jailbroken =
       bsg_get_map_value_bool(env, jni_cache, data, "jailbroken");
   event->device.total_memory =
       bsg_get_map_value_long(env, jni_cache, data, "totalMemory");
-  event->device.dpi = bsg_get_map_value_int(env, jni_cache, data, "dpi");
-  event->device.screen_density =
-      bsg_get_map_value_float(env, jni_cache, data, "screenDensity");
```

Comment:
This method is serializing metadata that is collected under the "device" tab. We send `device.time` as structured data and it can be found in serializer.c (L369)